File: JavaScript/primeNumber.js
function isPrime(n) {
    if (n < 2) return false;
    for (let i = 2; i < n; i++) {  // Should only check up to sqrt(n)
        if (n % i === 0) return false;
    }
    return true;
}

console.log(isPrime(997));  // Works but inefficient for large numbers

File: Python/primeNumber.py
def is_prime(n):
    if n < 2:
        return False
    for i in range(2, n):  # Inefficient loop (should check only up to √n)
        if n % i == 0:
            return False
    return True

print(is_prime(997))  # Works but is very slow for large numbers
